test/utils/base.py

```diff
@@ -232,36 +232,26 @@ def setUpClass(cls):
         # Safe to remove once https://github.com/Qiskit/qiskit-aer/pull/2179 is in a release version
         # of Aer.
         warnings.filterwarnings(
-            "default",
+            "ignore",
```

Comment:
I very very gently prefer `default` here personally, because it is a true warning (as opposed to the `__warningregistry__` one above, which is triggering falsely), and showing it means it's still at least _somewhat_ visible.

I don't feel strongly, though.

Reply:
reverted in cb1d6ef170ae85cd1399410d34f0bacb2f753a7e